pyproject.toml

```diff
@@ -21,13 +21,14 @@ toml = "^0.10"
 "ruamel.yaml" = "^0.16 || ^0.17"
 jinja2 = ">=2.11"
 jsonref = "^0.2"
-pydantic = "^1.6"
+pydantic = "^2.6"
```

Comment:
Do we need 2.6 as a minimum version for any particular reason or would `^2.0` suffice?

Reply:
I think you're right. Nothing that I can tell or see relies on any specific minor release of Pydantic and it would help it just be a more generic for looser requirements. Let me change that and retest.